packages/api/src/external/aws/cloudwatch.ts

```diff
@@ -4,7 +4,11 @@ import { capture } from "../../shared/notifications";
 
 export const METRICS_NAMESPACE = "Metriport";
 
-const cw = new AWS.CloudWatch({ apiVersion: "2010-08-01", region: Config.getAWSRegion() });
+// THINK ABOUT MORE
```

Comment:
So the mane issue with testing locally the AWS Stuff is that in order to get it to run we need to add an access key and secret key. So i was wondering if it makes sense to maybe do something like `process.env.ACCESSKEY ?? undefined` and then have a note above it stating why if that makes sense

Reply:
Are you sure we need access and secret key on the code? I just double tested OSS' `packages/util/test-s3.ts`, replacing the call to `s3.upload(...)` by a call to `s3.listBuckets()` and I got all the buckets we have. I'm assuming it uses the creds from AWS CLI (check w/ `aws sts get-caller-identity` and `aws s3 ls`, for example).

Reply:
also, the [question about prettier](https://github.com/metriport/metriport/pull/854/files#r1313624819)

Reply:
also, the [question about prettier](https://github.com/metriport/metriport/pull/854/files#r1313624819)

Yea this ill update, it was because i added the keys and it formatted.

Ill test what you suggest locally but how would this work when running from the GH

Reply:
> Ill test what you suggest locally but how would this work when running from the GH

[Check this out](https://github.com/metriport/metriport/blob/978a50ce57008feb946603baccbdbbda4b834b62/.github/workflows/_restart_api.yml#L53), we already set those on secrets, can just make sure CLI is being setup accordingly.